tests/qa_automation/kernel_kexec.pm

```diff
@@ -43,7 +43,8 @@ sub run {
     # kexec -l
     assert_script_run("kexec -l $kernel_new --initrd=$initrd_new --command-line='$cmdline'");
     # kexec -e
-    systemctl 'kexec';
+    # don't use built-in systemctl api, see poo#31180
+    type_string("systemctl kexec\n");
```

Comment:
Better use `script_run 'systemctl kexec', 0;` but rest looks fine

Reply:
@okurz I already tried, http://10.100.12.105/tests/825#step/kernel_kexec/33 , openqa is missing expected output on serial (which is obviously missing thanks to kernel boot)

Reply:
@okurz I did two more verifications http://10.100.12.105/tests/829 and http://10.100.12.105/tests/827 with script run and it didn't happen again. So it was probably one time glitch on my instance in job 825. Code was updated as was recommended.